roles/artemis/defaults/main.yml

```diff
@@ -68,7 +68,11 @@ artemis_war_url: "https://github.com/ls1intum/Artemis/releases/download/{{ artem
 
 artemis_system_ram_proportion: 0.8  # How much % of the system RAM to allocate to the Artemis service (Default here: 80%)
 
+artemis_database_schema: mysql
+artemis_database_type: MYSQL
```

Comment:
Can we use a template function here to convert the type to schema to upper case? https://jinja.palletsprojects.com/en/3.1.x/templates/#id11

Reply:
Done.